dev/setup-env.sh

```diff
@@ -3,7 +3,7 @@
 set -euo pipefail
 
 if [[ ! -d "venv" ]]; then
-    /usr/bin/python3.8 -m venv venv # use `sudo yum install python38` on Rocky Linux 8 to install this
+    /usr/bin/python3 -m venv venv # use `sudo yum install python38` on Rocky Linux 8 to install this
```

Comment:
Is this OK on both RL8 and RL9? I.e. we end up with a working python version which supports selinux?